More Robust SQL Server Section (#699)
OctopusDeploy/docs: docs/deployment-examples/database-deployments/index.md

### SQL Server

If you are deploying to SQL Server, see the [SQL Server section](/docs/deployment-examples/database-deployments/sql-server/index.md).

> Read them in the [SQL Server section](/docs/deployment-examples/database-deployments/sql-server/index.md).

Reviewer: ```suggestion
```

> If you are deploying to SQL Server, we have written a number of guides and blog posts on the subject.

Reviewer: ```suggestion
If you are deploying to SQL Server, see the [SQL Server section](/docs/deployment-examples/database-deployments/sql-server/index.md).
```
